src/morphology/morphoHelpers/buildFormFromRegex.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Checks if the input word qualifies for the input regex and if so builds a required form.
+ * This function is used for other more specific functions.
+ *
+ * @param {string} word The word to build forms for.
+ * @param {Array} regex The regex-based array of rules to compare the word against.
+ *
+ * @returns {string} The newly built form of the word.
+ */
+const buildOneFormFromRegex = function( word, regex ) {
```

Comment:
Kind of nit-picky, but could you rename the parameter `regex` to something else like `regexes`? 😅
I think the naming is a bit confusing now, because you're not passing a regex, but a collection of regexes.

Reply:
The whole debate on naming also got me thinking about this function. How about renaming it to `buildFormRule`? (and then `buildOneFormFromRule` & `buildTwoFormsFromRule`). Then we have a more logical order of first building rules from data, and then building forms with those rules.